### PR TITLE
build(bazel): add version stamping

### PR DESCRIPTION
**Pull request by jelbourn**
@alexeagle this captures setting the version on the packages we produce, but our existing build system also has a placeholder for the version of Angular we depend on (since that's also used across multiple packages). I'll probably have to do something manually to the output of `ng_package` to deal with that, since it's pretty custom.

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -3,9 +3,9 @@ workspace(name = "angular_material")
 # Add nodejs rules
 http_archive(
   name = "build_bazel_rules_nodejs",
-  url = "https://github.com/bazelbuild/rules_nodejs/archive/0.6.0.zip",
-  strip_prefix = "rules_nodejs-0.6.0",
-  sha256 = "e8a2bb5ca51fbafb244bc507bcebcae33a63d969f47413b319a8dcce032845bf",
+  url = "https://github.com/bazelbuild/rules_nodejs/archive/99166f8eb7fc628ca561acf9f9a51a1c26edadad.zip",
```
Comment (alexeagle) [inline WORKSPACE]: do you need this labelled? 0.7.0 was from Friday and should have what you need

File: src/lib/BUILD.bazel [hunk 2]
```diff
@@ -32,23 +32,20 @@ sass_bundle(
...
-#  ],
-#  packages = ["//src/lib/schematics:npm_package"],
-#  deps = MATERIAL_TARGETS
-#)
+ng_package(
+  name = "npm_package",
+  srcs = ["package.json"],
+  entry_point = "src/lib/public_api.js",
+  entry_point_name = "material",
+  globals = ROLLUP_GLOBALS,
```
Comment (alexeagle) [inline src/lib/BUILD.bazel]: check if the globals are out of date now that rxjs has only two entry points

File: tools/bazel.rc [hunk 3]
```diff
@@ -36,8 +36,9 @@ test --test_output=errors
 build --experimental_ui
 test --experimental_ui
 
-# Output to a more coventional location
-build --symlink_prefix=dist/bazel/
+# Configures script to do version stamping.
```
Comment (alexeagle) [inline tools/bazel.rc]: do you mean to remove symlink_prefix?

File: WORKSPACE (hunk 1 above)
Comment (jelbourn) [reply WORKSPACE]: It doesn't really need to be, I can update it next time around

File: src/lib/BUILD.bazel (hunk 2 above)
Comment (jelbourn) [reply src/lib/BUILD.bazel]: They should be good; this just captures the globals specific to our own packages

File: tools/bazel.rc (hunk 3 above)
Comment (jelbourn) [reply tools/bazel.rc]: Yeah, I meant to remove it a while ago and lost the change

**Pull request merged**